isabelle/Core/SingleInputTransactions.thy

```diff
@@ -1234,5 +1226,5 @@ lemma inputToTransactions_isSingleInput : "isSingleInput (inputsToTransactions s
 lemma traceListToSingleInput_isSingleInput2 : "isSingleInput (traceListToSingleInput t)"
```

Comment:
`traceListToSingleInput_isSingleInput2` is unused in the rest of the code base as well